src/item.rs

```diff
@@ -83,7 +83,7 @@ impl<'a> Item {
 
         let mut ret = Item {
             index: index,
-            output_text: orig_text,
+            orig_text: orig_text,
```

Comment:
```suggestion
            orig_text,
```

Reply:
In recent rust version, it is better to omit the value variable if the attribute and value variable have the same name.

Perhaps another commit removing unnecessary variable names? Thanks!